lib/publiccloud/ssh_interactive.pm

```diff
@@ -22,13 +22,14 @@ our @EXPORT = qw(ssh_interactive_tunnel ssh_interactive_leave select_host_consol
 sub establish_tunnel_console {
     my ($console) = @_;
 
-    select_console("$console");
+    select_console("$console", timeout => 180);
```

Comment:
Did you try without this?

Reply:
No, I Just think the code change is no harm or even better.

Reply:
Let's see what other say about it. I'm not sure.